### PR TITLE
(#522) App should serve a "Page Not Found" page when the view page is passed an invalid/nonexistent trial ID

### PR DESCRIPTION
**Pull request by alerosad**
- added page not found message, based on the current drupal configuration for a similar case

### CODE REVIEW

**Comment by alinai** [comment]
@alerosad, Please let me know where I am testing this one. Thanks!

**Comment by alinai** [comment]
Looks good on https://react-app-dev.cancer.gov/clinical-trials-search-app/pr-529. Thanks @alerosad

**Comment by sarinapadilla** [comment]
@mworrest can you take a look at this and confirm it looks good? Thanks!

**Review by mworrest — APPROVED** [review]
Looks good to me.

**Comment by arcepaul** [comment]
There is no need to worry about spanish versions for CTS.  Please remove the utilities and other things that make that portion up.

File: src/views/PageNotFound/PageNotFound.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+const PageNotFound = () => {
+	const [{ canonicalHost }] = useAppSettings();
+	const [searchText, updateSearchText] = useState('');
+	const tracking = useTracking();
+
+	const i18n = {
+		errorPageText: 'An error occurred. Please try again later.',
+		noResultsFoundFor: '0 results found for',
+		pageNotFoundTitle: 'Page Not Found',
+		pleaseCheckSpellingOrTryAnotherSearch:
```
Comment (arcepaul) [inline src/views/PageNotFound/PageNotFound.jsx:17]: is this unused?

**Review by arcepaul — APPROVED** [review]

**Pull request closed without merging**